fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -525,6 +535,21 @@ public void close() {
         }
     }
 
+    private void maybeLogBuildProgress(Subspace subspace, Tuple startTuple, Tuple endTuple, Tuple realEnd) {
+        if (LOGGER.isInfoEnabled()
+                && (progressLogIntervalMillis >= 0
+                    && System.currentTimeMillis() - timeOfLastProgressLogMillis >= progressLogIntervalMillis)) {
+            LOGGER.info(KeyValueLogMessage.of("Built Range",
+                    "indexName", index.getName(),
+                    "indexVersion", index.getLastModifiedVersion(),
+                    "subspace", subspace,
```

Comment:
Probably want either `ByteArrayUtil2.loggable(subspace.getKey())` if it's not going to also pass, say, the `SubspaceProvider` into `handleBuiltRange` to get proper subspace logging.

I think some of these keys (index name, for example) have entries in `LogMessageKeys` that should be used instead of literal strings.